src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public static final int RANDOM_SIZE = 10;
+    public static final int FIGURES_AMOUNT = 5;
+
+    public Figure getRandomFigure() {
+        String nameCircle = "circle";
+        int randomR = (int) (Math.random() * RANDOM_SIZE) + 1;
+        ColorSupplier colorFigure = new ColorSupplier();
+        Figure circle = new Circle(nameCircle,
+                colorFigure.getRandomColor(), randomR);
```

Comment:
```suggestion
public class FigureSupplier {
    public static final int RANDOM_SIZE = 10;
    public static final int FIGURES_AMOUNT = 5;
    private ColorSupplier colorSupplier = new ColorSupplier();

    public Figure getRandomFigure() {
        String nameCircle = "circle";
        int randomR = (int) (Math.random() * RANDOM_SIZE) + 1;
        Figure circle = new Circle(nameCircle,
                colorFigure.getRandomColor(), randomR);
```
Сервісні класи краще оголошувати як поля класу а не локальні. Щоб в тебе створювався один об'єкт ColorSupplier на клас а не на кожен виклик методу getRandomFigure